examples/crna-kitchen-sink/storybook/stories/Welcome/index.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+
+import React, { PropTypes } from 'react';
+import { View, Text } from 'react-native';
+
+export default class Welcome extends React.Component {
+  styles = {
+    wrapper: {
+      flex: 1,
+      padding: 24,
+      justifyContent: 'center',
+    },
+    header: {
+      fontSize: 18,
+      marginBottom: 18,
+    },
+    content: {
+      fontSize: 12,
+      marginBottom: 10,
+      lineHeight: 18,
+    },
+  };
+
+  showApp(event) {
+    event.preventDefault();
```

Comment:
On third thought, I think this is a separate issue. This used to be used to show off `addon-links` but was removed at some point. Opened: https://github.com/storybooks/storybook/issues/1569